graphics/src/AssimpLoader.cc

```diff
@@ -307,6 +306,7 @@ Mesh *AssimpLoader::Load(const std::string &_filename)
       aiProcess_RemoveRedundantMaterials |
       aiProcess_SortByPType |
       aiProcess_Triangulate |
+      aiProcess_GenNormals |
```

Comment:
Do you have an example asset where this happens?

Reply:
Adding it below: 
[airbot.zip](https://github.com/gazebosim/gz-common/files/8751580/airbot.zip)